rosdep/python.yaml

```diff
@@ -620,6 +622,7 @@ python-defusedxml:
   debian: [python-defusedxml]
   fedora: [python-defusedxml]
   gentoo: [dev-python/defusedxml]
+  opensuse: [python3-defusedxml]
```

Comment:
Is that available in a default installation? I don't seem to have instructions for OpenSUSE here to test: http://wiki.ros.org/ROS/Installation